src/ILLink.Tasks/CreateRuntimeRootDescriptorFile.cs

```diff
@@ -132,9 +137,64 @@ void ProcessNamespaces (string namespaceFile)
 		void ProcessMscorlib (string typeFile)
```

Comment:
Should any of the other files get the same treatment? rexcep.h has a few conditional ones:
https://github.com/dotnet/runtime/blob/58b25dfa8fae5f5a382a6ffbe89e9aee479eb31c/src/coreclr/vm/rexcep.h#L181-L183

Reply:
Good point.